go.mod

```diff
@@ -74,11 +73,14 @@ require (
 	google.golang.org/protobuf v1.29.1 // indirect
 )
 
+require github.com/hashicorp/golang-lru/arc/v2 v2.0.5
```

Comment:
can you move this back to the direct deps `require` stanza? Somehow it ended up out of it...

Reply:
Not sure why that happened but cleaned it up.